index.html

```diff
@@ -14,7 +14,7 @@
       <div class="container">
         <nav id="nav">
           <div class="Logo">
-            <span>Ashour</span>
+            <span>My Logo</span>
           </div>
```

Comment:
- [ ] It would be better is remove the `<span>` tag and use the `<a>` link tag for semantic reasons and HTML best practices.
- [ ] OPTIONAL: You can also change the logo name **My Logo** to your name, so it will look more beautiful as it is a personal portfolio.
- [ ] Please, put the `logo` in the header but outside of the `nav` tag to ensure that your `nav` only contains navigation links.
- [ ] As suggested by the last reviewer, ensure that colours and image size match with the Figma. Kindly match your `menu` hamburger as well.

1. **Yours**

![image](https://user-images.githubusercontent.com/80612925/170340604-a7b4b16f-c869-47e0-8b14-de757aedf540.png)

2. **Microverse Template**

![image](https://user-images.githubusercontent.com/80612925/170341084-a55bd450-5b72-46bd-87c4-fcf5dc8e8a16.png)